packages/playground/src/App.tsx

```diff
@@ -1,15 +1,18 @@
-import { Layout } from './components/Layout';
-import { Header } from './components/Header';
-import { Editor } from './components/Editor';
-import { Console } from './components/Console';
+import { Layout } from "~/components/Layout";
```

Comment:
`components` should have an `index.ts` file from which the Components are imported.